freeipa/src/main/java/com/sequenceiq/freeipa/client/ClusterProxyErrorRpcListener.java

```diff
@@ -0,0 +1,71 @@
+package com.sequenceiq.freeipa.client;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.google.common.annotations.VisibleForTesting;
+import com.googlecode.jsonrpc4j.JsonRpcClient;
+import com.sequenceiq.cloudbreak.clusterproxy.ClusterProxyException;
+
+import java.util.Optional;
+
+public class ClusterProxyErrorRpcListener implements JsonRpcClient.RequestListener {
+
+    private static Optional<ClusterProxyError> deserializeAsClusteProxyError(ObjectNode objectNode) {
```

Comment:
why is this method static?

Reply:
To indicate that it doesn't change the internal state of object.  I'll remove the static since I don't have a strong preference.